plugins/kubernetes/app/models/concerns/kubernetes/api/deleted_pod.rb

```diff
@@ -0,0 +1,9 @@
+module Kubernetes
+  module Api
+    module DeletedPod
```

Comment:
Fair enough. :)

I actually would like to get @henders opinion on this. I'm dynamically extending the Pod objects with this module when we get a deleted Pod in the DeployWatcher. Is this an acceptable pattern in Ruby?